ftwhttp/header_values/constants.go

```diff
@@ -0,0 +1,5 @@
+package ftwhttp
+
+const (
+	ApplicationXWwwFormUrlencoded = "application/x-www-form-urlencoded"
+)
```

Comment:
Ok. Maybe creating a `header` package instead? And moving all constants there?

Reply:
My problem in Go is namespacing. If both names and values are in the same package, then we need to add some prefix to every entry to make it clear what the constant is. Using distinct packages solves this namespacing problem, even though it's pretty ugly.

Reply:
Yeah. We cannot have both. Or we have the prefixed version, or we have ugly namespaces.

Reply:
I asked @M4tteoP for his opinion on this and he agreed that the way it is now it's at least readable. He didn't see a better option either. Should we leave it this way?

Reply:
Ping @fzipi.